src/storage/src/hummock/local_version.rs

```diff
@@ -103,20 +109,25 @@ impl SyncUncommittedData {
 impl LocalVersion {
     pub fn new(
         version: HummockVersion,
-        unpin_worker_tx: UnboundedSender<HummockVersionId>,
+        pinned_version_manager_tx: UnboundedSender<PinVersionAction>,
     ) -> Self {
-        let mut version_ids_in_use = BTreeSet::new();
-        version_ids_in_use.insert(version.id);
+        if pinned_version_manager_tx
```

Comment:
Each `Pin` should be associated with a `PinnedVersionGuard`. Upon creation of a `PinnedVersionGuard`, we send a `Pin` to the tx and upon drop we send `Unpin` so that the management gets easier.